### PR TITLE
ffmpeg 3.1.1

### PR DESCRIPTION
**Pull request by zmwangx**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

https://github.com/FFmpeg/FFmpeg/releases/tag/n3.1.1.
- (Update) Added option with-libebur128, introduced in 3.1 and requested in
  #2556;
- Removed the pnm_parser patch over 3.1, which has shipped;
- Removed the option with-dcadec, which has in fact been removed in
  3.1 (see https://github.com/FFmpeg/FFmpeg/blob/n3.1/Changelog).

### CODE REVIEW

**Comment by ilovezfs** [comment]
@zmwangx `args << "--enable-libdcadec" if build.with? "dcadec"`

**Comment by zmwangx** [comment]
That was stupid, sorry.

**Comment by ilovezfs** [comment]
No need to apologize. I've already deducted two gold stars, so we're even.

**Comment by ilovezfs** [comment]
seems mpv needs a little bump again

**Comment by zmwangx** [comment]
Bumped mpv, and added `with-libebur128` as requested in #2556.

File: Formula/ffmpeg.rb [hunk 1]
```diff
@@ -42,11 +32,11 @@ class Ffmpeg < Formula
   option "with-webp", "Enable using libwebp to encode WEBP images"
   option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
   option "with-snappy", "Enable Snappy library"
-  option "with-dcadec", "Enable dcadec library"
   option "with-rubberband", "Enable rubberband library"
   option "with-zimg", "Enable z.lib zimg library"
   option "with-openh264", "Enable OpenH264 library"
   option "with-xz", "Enable decoding of LZMA-compressed TIFF files"
+  option "with-libebur128", "Enable libebur128 for EBU R128 loudness measurement"
```
Comment (zmwangx) [inline Formula/ffmpeg.rb]: I derived the option description from FFmpeg's `configure`; now that I look at existing option descriptions though, "Enable *using* libebur128 for EBU R128 loudness measurement" could be better. But this is just a triviality.

Comment (ilovezfs) [inline Formula/ffmpeg.rb]: Fixed in post.

**Comment by ilovezfs** [comment]
@zmwangx Thanks for fixing this so promptly! Your two :star2: have been restored, plus an additional one.

**Pull request closed without merging**